libs/mpi/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+# if networking is off, or MPI parcelport is not being used
+# then allow the option of using an MPI
+if ("${HPX_WITH_NETWORKING}" AND "${HPX_WITH_PARCELPORT_MPI}")
+  hpx_option(HPX_WITH_MPI_FUTURES BOOL
+    "Enable support for returning futures from MPI asynchronous calls"
+    " (default: ON if networking+MPI already enabled)"
+    ON CATEGORY "MPI")
+else()
+  hpx_option(HPX_WITH_MPI_FUTURES BOOL
+    "Enable support for returning futures from MPI asynchronous calls"
+    " (default: ON if networking+MPI already enabled)"
+    OFF CATEGORY "MPI")
+endif()
```

Comment:
What's the difference between the two options (except for the default value)?

Reply:
We usually name module-specific configuration constants as `HPX_<modulename>_WITH_<option>`. Could you please adapt this?